README.md

```diff
@@ -36,6 +57,15 @@ Change directory to `src/Ztm.Data.Entity.Postgres` then run:
 ZTM_MAIN_DATABASE="Host=127.0.0.1;Database=ztm;Username=ztm" dotnet ef database update
 ```
 
+## Start Zcoin Daemon
+
+Grab latest stable Zcoin binary from [here](https://github.com/zcoinofficial/zcoin/releases), extract or install it,
+then start Zcoin daemon:
+
+```sh
+./zcoind -printtoconsole -rpcuser=zcoin -rpcpassword=zcoin
```

Comment:
should suggest user to don't use default password. or if you need this one as dev environment setup guide please use testnet instead.